### DISCUSSION TITLE
Grid-template-columns arbitrary values not working?

### DISCUSSION THREAD

**Discussion by Daguima-dag** [tailwindlabs/tailwindcss#6649] [Help]
Hi! I tried to pass an arbitrary value to a `grid-template-columns` utility (`grid-cols-[...]`) but the output still has the underscores. What could be the reason?
For example: `grid-cols-[auto_auto]` outputs `grid-template-columns: auto_auto`.

**Comment by adamwathan** [comment]
Hey! This works as expected in my testing:

https://play.tailwindcss.com/zGj9y3IKRz

![image](https://user-images.githubusercontent.com/4323180/146823588-385446f9-c8ae-4760-8c07-485274d27134.png)
